src/components/Login.js

```diff
@@ -9,35 +9,36 @@ class Login extends Component {
       password: ''
     };
   }
+
   handleChange(event) {
     this.setState({
       [event.target.name]: event.target.value
     })
     event.preventDefault();
   }
-  handleSubmit(event) {
-     // Connect to backend
-     alert(`E-mail: ${this.state.email}, Password: ${this.state.password}`);
+  handleSubmit=(event)=>{
     event.preventDefault();
+    this.props.onSubmit(this.state);
   }
+
   render() {
     return (
       <div className="container">
         <h3>Login to...</h3>
         <div className="row">
-          <form className="col s12" onSubmit={event => this.handleSubmit(event)}>
+          <form className="col s12" onSubmit={(event) => this.handleSubmit(event)}>
```

Comment:
I am not quite sure why you added the `()`  braces.

Reply:
🤦‍♂️ I'm not sure why I did that either. I will remove them and merge this branch.